### DISCUSSION TITLE
threshold.conf

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#2159] [Unsupported Versions] [answered]
Can I just put my threshold configs in the threshold.conf file?  I know the docs say use the pillar but I'm just looking at an alternate method.  I find it easier and it's just like version 16 that way.

**Comment by TOoSmOotH** [comment, marked as answer]
Currently salt builds this file so you cannot modify it directly. You must modify the pillar to get this functionality.

**Reply by gcsrellis** [reply]
Does the pillar still have the ability to reference the variables in suricata.yaml?  Say for instance like this
thresholding:
  sids:
    2000488:
      - suppress:
          gen_id: 1
          track: by_src
          ip: $DNS_SERVERS
